285610_robot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint_r_f_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel_1_joint
    pid: {p: 10.0, i: 0.001, d: 1.0}

  joint_r_b_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel_2_joint
    pid: {p: 10.0, i: 0.001, d: 1.0}

  joint_l_f_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel_3_joint
    pid: {p: 10.0, i: 0.001, d: 1.0}

  joint_l_b_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: wheel_4_joint
    pid: {p: 10.0, i: 0.001, d: 1.0}

  joint_turret_position_controller:
    type: effort_controllers/JointPositionController
    joint: upper_arm_joint
    pid: {p: 10.0, i: 0.001, d: 1.0}

  joint_forearm_position_controller:
    type: effort_controllers/JointPositionController
    joint: forearm_joint
    pid: {p: 10.0, i: 0.001, d: 1.0}

  joint_prism_position_controller:
    type: velocity_controllers/JointPositionController
    joint: prism_part_joint
    pid: {p: 10.0, i: 0.001, d: 1.0}